google-cloud-language/Rakefile

```diff
@@ -61,10 +66,32 @@ end
 
 # Acceptance tests
 desc "Run the google-cloud-language acceptance tests."
-task :acceptance do
+task :acceptance, :project, :keyfile do |t, args|
+  project = args[:project]
+  project ||= ENV["GCLOUD_TEST_PROJECT"] || ENV["LANGUAGE_TEST_PROJECT"]
+  keyfile = args[:keyfile]
+  keyfile ||= ENV["GCLOUD_TEST_KEYFILE"] || ENV["LANGUAGE_TEST_KEYFILE"]
+  if keyfile
+    keyfile = File.read keyfile
+  else
+    keyfile ||= ENV["GCLOUD_TEST_KEYFILE_JSON"] || ENV["LANGUAGE_TEST_KEYFILE_JSON"]
+  end
+  if project.nil? || keyfile.nil?
+    fail "You must provide a project and keyfile. e.g. rake acceptance[test123, /path/to/keyfile.json] or LANGUAGE_TEST_PROJECT=test123 LANGUAGE_TEST_KEYFILE=/path/to/keyfile.json rake acceptance"
+  end
+  # always overwrite when running tests
+  ENV["LANGUAGE_PROJECT"] = project
+  ENV["LANGUAGE_KEYFILE"] = nil
+  ENV["LANGUAGE_KEYFILE_JSON"] = keyfile
+
+  # Required for smoke tests
+  ENV["SMOKE_TEST_PROJECT"] = project
+
   Rake::Task["smoke_test"].invoke
 end
 
+
+
 namespace :acceptance do
   task :run do
```

Comment:
This task seems out of place. Is the intention that the `acceptance` task is supposed to call `acceptance:run`?

Reply:
Yes, that's right, this is a code generator bug. Both these tasks are currently no-ops right now, but will fix this in the next refresh. Tracking as https://github.com/googleapis/toolkit/issues/1688.